src/role-access/components/choose-radio-option/choose-radio-option.component.ts

```diff
@@ -11,7 +10,7 @@ import { OptionsModel } from '../../models/options-model';
 export class ChooseRadioOptionComponent {
 
   @Input() public title: string;
-  @Input() public caption: RoleAllocationCaptionText;
+  @Input() public caption: string;
```

Comment:
Minor query - Why is this change necessary?

Reply:
this is to make caption to be dynamic, it's not only for specific enum.

Reply:
ok sure